client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
+
+import (
+	"github.com/go-redis/redis"
```

Comment:
I agree with Dave's comment about this file's package name - I would go with `package redis`
and then, for this line, use: 
```suggestion
	goredis "github.com/go-redis/redis"
```

to differentiate the imported package from this one

Reply:
Didn't think of using an import alias.